stdlib-bootstrapped/src/scala/reflect/Manifest.scala

```diff
@@ -173,7 +173,7 @@ object ManifestFactory {
 
   @SerialVersionUID(1L)
   final private[reflect] class ByteManifest extends AnyValManifest[scala.Byte]("Byte") {
-    def runtimeClass = java.lang.Byte.TYPE
+    def runtimeClass: Class[java.lang.Byte] = java.lang.Byte.TYPE
```

Comment:
This is annoying because it is not actually correct. The Java signature of `Byte.TYPE` says it returns a `Class<java.lang.Byte>`, but it actually returns the class for the primitive type `byte`. In Scala, that should be typed as `Class[scala.Byte]` instead.

Since this lives in a `private[reflect]` class, I suggest we keep ignoring it for now, and instead send a patch upstream to declare it as
```suggestion
    def runtimeClass: Class[_] = java.lang.Byte.TYPE
```
The same applies to all the other changes in this file.